spec/lib/tacos_demo.py

```diff
@@ -213,25 +213,15 @@ def parse_comments(
 def parse_comment(
     job_filter: str | None, slices_subpath: Path, comment: str
 ) -> Generator[tuple[str, Slice, str]]:
-    remainder = comment
-    while True:
-        comment, tag_start, remainder = remainder.partition(COMMENT_TAG)
-        if not tag_start:
-            return
-
-        tag, tag_end, remainder = remainder.partition(COMMENT_TAG_END)
-
-        tag = Parse(tag)
-        job = tag.before.first("(")  # )
+    lines = comment.splitlines(keepends=True)
+    points = [i for i, l in enumerate(lines) if COMMENT_TAG in l]
+    points.append(len(lines))
```

Comment:
what does points represent exactly? is that the indexes with the total line count appended on the list?

Reply:
points is the delimiting indexes of the individual sections in a combined comment.